spec/models/user_spec.rb

```diff
@@ -55,25 +82,37 @@
   end
 
   describe 'unify' do
-    let(:user_1) {FactoryGirl.create(:user, user_name: 'Thomas')}
-    let(:user_2) {FactoryGirl.create(:user, user_name: 'Anders')}
-    let(:user_3) {FactoryGirl.create(:user, user_name: 'Kalle')}
+
+    let(:user_1) { FactoryGirl.create(:user, user_name: 'Thomas', mentor: true) }
+    let(:user_2) { FactoryGirl.create(:user, user_name: 'Anders') }
+    let(:user_3) { FactoryGirl.create(:user, user_name: 'Kalle') }
+    let(:user_4) { FactoryGirl.create(:user, user_name: 'Sam', mentor: true) }
 
     before do
       user_1.skill_list.add('java-script, testing, ruby', parse: true)
       user_2.skill_list.add('java-script, java, html', parse: true)
       user_3.skill_list.add('java, html', parse: true)
+      user_4.skill_list.add('testing, ruby', parse: true)
       user_1.save
       user_2.save
       user_3.save
+      user_4.save
```

Comment:
These saves are unnecessary. When you do `skill_list.add` it adds the skills and saves them as well.